### PR TITLE
tools: improve DM deployment and config docs

### PR DESCRIPTION
**Pull request by lilin90**
This PR updated DM docs based on the following PR and DM docs testing.

Related PR: https://github.com/pingcap/tidb-tools/pull/169

@GregoryIan @csuzhangxc @queenypingcap @CaitinChen PTAL

### CODE REVIEW

File: tools/dm/deployment.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Before you start, make sure you have the following machines as required.
     > **Note:** The Control Machine can be one of the target machines.
 
     - CentOS 7.3 (64-bit) or later, with Python 2.7 installed
+    - Ansible 2.5 or later installed
```
[the remark this answers is not in the document]
Comment (lilin90) [reply tools/dm/deployment.md]: No, from my DM testing.

[the remark this answers is not in the document]
Comment (CaitinChen) [reply tools/dm/deployment.md]: OK. Cool!

File: tools/dm/deployment.md [hunk 2]
```diff
@@ -181,29 +184,31 @@ You can choose one of the following two types of cluster topology according to y
 
 - [The cluster topology of a single DM-worker instance on each node](#option-1-use-the-cluster-topology-of-a-single-dm-worker-instance-on-each-node)
 
-    Generally, it is recommended to deploy one DM-worker instance on each node. Howerver, if the CPU and memory of your machine is much better than the required in [Hardware and Software Requirements](/op-guide/recommendation.md), and you have more than 2 disks in one node or the capacity of one SSD is larger than 2 TB, you can deploy no more than 2 DM-worker instances on a single node.
-
 - [The cluster topology of multiple DM-worker instances on each node](#option-2-use-the-cluster-topology-of-multiple-dm-worker-instances-on-each-node)
 
+    Generally, it is recommended to deploy one DM-worker instance on each node. However, if the CPU and memory of your machine is much better than the required in [Hardware and Software Requirements](/op-guide/recommendation.md), and you have more than 2 disks in one node or the capacity of one SSD is larger than 2 TB, you can deploy no more than 2 DM-worker instances on a single node.
```
[the remark this answers is not in the document]
Comment (lilin90) [reply tools/dm/deployment.md]: Yes.

**Comment by lilin90** [comment]
@GregoryIan PTAL

**Pull request merged**